src/utils/validators.ts

```diff
@@ -1,6 +1,12 @@
 import fs from 'fs'
 import path from 'path'
 
+export const toDate = (val: any): Date | null => {
```

Comment:
Any particular reason this got placed in here?

Reply:
no.. not sure how this got in here. I thought it was b/c i merged master, but i don't see it in there either. removed